integration/ctr/oci_hooks_tests.bats

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env bats
+#
+# Copyright (c) 2022 Apple Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+IMAGE="${IMAGE:-docker.io/library/busybox:latest}"
```

Comment:
It would be nice if you could use an image from quay.io or a registry other than docker.io. As people behind a firewall can very often hit the limit of pull requests imposed by docker.io

Reply:
thanks for this suggestion, I didn't consider the limits with docker.io.

I see `quay.io/prometheus/busybox:latest` used in a few places in the repo. I have tested it locally -- ill push that up as a change, but if you have a more opinionated recommendation happy to try it out.